### ISSUE TITLE
Publish to pypi

### ISSUE THREAD

**Issue by andybergon** [amas0/raiderio#1]
Maybe using GH Actions.

I can take care of it if you want.

@amas0

**Comment by amas0** [comment]
Thanks for raising -- added to PyPi. Install instructions in README from commit f776924040ab50259e42141b3e7e64211e76aab7

**Comment by andybergon** [comment]
How did you publish it?

The wheel seems to be empty:
```sh
> pip download --no-deps raiderio
> python -m zipfile -l raiderio-1.0-py3-none-any.whl

File Name                                             Modified             Size
raiderio-1.0.dist-info/METADATA                2021-09-05 15:35:22          210
raiderio-1.0.dist-info/WHEEL                   2021-09-05 15:35:22           92
raiderio-1.0.dist-info/top_level.txt           2021-09-05 15:35:22            1
raiderio-1.0.dist-info/RECORD                  2021-09-05 15:35:22          292

```

**Comment by amas0** [comment]
Thanks for catching this. I seem to have made a mistake in my `setup.py` that I didn't catch at first which caused the wheel to build without the actual source inside of it. I believe I fixed it in 0c756a1a323eec131bbc2088707ca69ecf9f4459. I uploaded a new version 1.0.1 to pypi with this change and confirmed after download that it looks correct:

```
amas ~/projects/raiderio $ python -m zipfile -l raiderio-1.0.1-py3-none-any.whl
File Name                                             Modified             Size
raiderio/__init__.py                           2020-09-20 22:06:34           26
raiderio/api.py                                2021-09-05 15:33:42         5286
raiderio-1.0.1.dist-info/METADATA              2022-03-05 20:00:14          193
raiderio-1.0.1.dist-info/WHEEL                 2022-03-05 20:00:14           92
raiderio-1.0.1.dist-info/top_level.txt         2022-03-05 20:00:14            9
raiderio-1.0.1.dist-info/RECORD                2022-03-05 20:00:14          447
```
